login-workflow/src/translations/i18n.ts

```diff
@@ -7,12 +7,20 @@ void i18n
     .use(initReactI18next) // passes i18n down to react-i18next
     .init({
         resources: {
+            // English
             en: translations.english,
+            // French
             fr: translations.french,
             fr_US: translations.french,
             fr_CA: translations.french,
+            fr_FR: translations.french,
+            // Spanish
+            es: translations.spanish,
+            es_US: translations.spanish,
+            es_MX: translations.spanish,
+            es_ES: translations.spanish,
         },
-        lng: 'en', // TODO get the device locale for the default
+        lng: 'es', // TODO get the device locale for the default
```

Comment:
does this mean we are using `es` by default

Reply:
Yeah, good catch. That's going to change to using the browser default in the commit that I'll be adding to this tomorrow.